### PR TITLE
Refs #36723 - Release note for Fedora/RH Atomic removal

### PR DESCRIPTION
**Pull request by ekohl**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: guides/doc-Release_Notes/topics/foreman.adoc [hunk 1]
```diff
@@ -7,5 +7,11 @@
 [id="foreman-upgrade-warnings"]
 == Upgrade Warnings
 
+=== Fedora/Red Hat Atomic support dropped
+
+Red Hat Atomic went end of life in 2021 and Fedora even earlier.
+Starting Foreman 3.9 these will be recognized as regular Fedora or Red Hat.
```
Comment (maximiliankolb) [inline guides/doc-Release_Notes/topics/foreman.adoc:13]:
```suggestion
Starting with Foreman 3.9, these will be recognized as regular Fedora or Red Hat.
```

File: guides/doc-Release_Notes/topics/foreman.adoc [hunk 2]
```diff
@@ -7,5 +7,11 @@
 [id="foreman-upgrade-warnings"]
 == Upgrade Warnings
 
+=== Fedora/Red Hat Atomic support dropped
+
+Red Hat Atomic went end of life in 2021 and Fedora even earlier.
```
Comment (maximiliankolb) [inline guides/doc-Release_Notes/topics/foreman.adoc:12]:
```suggestion
Red Hat Atomic went end of life in 2021 and Fedora Atomic even earlier.
```

**Review by maximiliankolb — COMMENTED** [review]
rest LGTM

**Review by maximiliankolb — APPROVED** [review]

File: guides/doc-Release_Notes/topics/foreman.adoc [hunk 3]
```diff
@@ -7,5 +7,11 @@
 [id="foreman-upgrade-warnings"]
 == Upgrade Warnings
 
+=== Fedora/Red Hat Atomic support dropped
```
Comment (mjivraja) [inline guides/doc-Release_Notes/topics/foreman.adoc:10]:
```suggestion
=== Fedora/Red Hat Atomic is no longer supported
```
Can this be better?

Comment (ekohl) [reply guides/doc-Release_Notes/topics/foreman.adoc:10]: Is that really better? I always think headers should be short.

Looking at writers here.

Comment (mjivraja) [reply guides/doc-Release_Notes/topics/foreman.adoc:10]: When we drop things, we mostly use `no longer supported`.

**Review by mjivraja — APPROVED** [review]

**Pull request merged**